app/javascripts/helpers/web3Service.jsx

```diff
@@ -17,9 +30,9 @@ class Web3Service {
     this.slotManagerContract = null;
     this.slotStorageContract = null;
     this.storageAddr = null;
-    this.myOccupiedGameInitWatchers = {};
+    this.myInitializedGameInitWatchers = {};
 
-    if (typeof web3 !== 'undefined') {
+    if (typeof web3 === 'undefined') {
```

Comment:
it causes error when user doesn't have metamask plugin. you should fix it to "!==".